CommunityToolkit.Diagnostics/Attributes/CallerArgumentExpressionAttribute.cs

```diff
@@ -0,0 +1,31 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+#if !NET5_0_OR_GREATER
+
+namespace System.Runtime.CompilerServices;
+
+/// <summary>
+/// An attribute that allows parameters to receive the expression of other parameters.
+/// </summary>
+/// <remarks>Internal copy from the BCL attribute.</remarks>
```

Comment:
This basically just makes the attribute a no-op to compile for .NET Standard 2.0, eh?

Do we need to worry about if folks drop the argument on non .NET 5+ platforms? Is that something we can analyze or provide info on somewhere?

Reply:
We don't need to worry, this feature will work regardless of target runtime as long as you're using VS2022+, since it's purely a Roslyn feature, it's just syntax sugar really. This means it will also work on eg. .NET Standard 2.0, UWP, etc. just fine. This attribute is just needed because Roslyn looks it up and matches it by name when compiling code, but it doesn't actually need to be part of the runtime itself (as in, it's like the nullability attributes that can be polyfilled in the same way, not like eg. `[StackTraceHidden]` which needs explicit runtime support and is matched by identity and not name).